CoinGeckoMarketCap.cs

```diff
@@ -78,11 +80,14 @@ public override BaseData Reader(SubscriptionDataConfig config, string line, Date
             var csv = line.Split(',');
 
             var parsedDate = Parse.DateTimeExact(csv[0], "yyyyMMdd");
+
+	    decimal marketcap = decimal.Parse(csv[1], NumberStyles.Any, CultureInfo.InvariantCulture);
             return new CoinGeckoMarketCap
             {
                 Symbol = config.Symbol,
-                SomeCustomProperty = csv[1],
-                Time = parsedDate - Period,
+                Time = parsedDate,
```

Comment:
Is this the right behavior? Since `EndTime = Time + Period`, will this result into forward-bias? Suppose data receiving at `2020-01-01 00:00`  is having the `EndTime = DateTime(2020, 1, 1, 0, 0)` and representing the market cap of `2019-12-31`.